core/src/main/java/org/mule/runtime/core/internal/execution/ModuleFlowProcessingPhase.java

```diff
@@ -94,6 +95,7 @@
 public class ModuleFlowProcessingPhase
     extends NotificationFiringProcessingPhase<ModuleFlowProcessingPhaseTemplate> implements Initialisable {
 
+  private static final String FLOW_BACK_PRESSURE_ERROR_TYPE_NOT_FOUND = "FLOW_BACK_PRESSURE error type not found";
```

Comment:
who uses this?

Reply:
Think i added it in the past for testing. Just removed it.